web/themes/custom/cloudy/templates/field/field-group-html-element.html.twig

```diff
@@ -20,9 +20,7 @@
 {% set classes = classes|default([]) %}
 
 {%
-  set group_classes = [
```

Comment:
The margin variable value is still set in the field group date and time templates. By removing this, we are relying on fields to set margin when it should be the containing element that controls the margin. Also, having unused variables in the group templates is not ideal.

Reply:
Every usage we have has a fields for date and time. After standardizing the field spacing every instance looked a bit weird because of the additional top and bottom spacing on the field group. Hence the removal. I didn't clean up the passing of margin from the group-date, group-project-date, and group-time. I've done that now.

I'm definitely interested in the refactor of all this date logic. Even when we originally rolled this out, I didn't necessarily care for theming the field group to achieve the output. It has a lot of drawbacks. (For instance, that method won't work if we move to Layout Builder.) I couldn't think of a better way to do it at the time, but I'm kinda keen on the Smart Date approach going forward as it would really simplify a lot of this logic. 😄 

I went ahead and removed the margin variables in this next commit.

Reply:
Yes, the datetime fields have been overly complex, making theming difficult.